README.md

```diff
@@ -0,0 +1,144 @@
+| `service.type`                       | k8s service type (e.g. NodePort, LoadBalancer)  | `NodePort`                                                 |
+| `service.port`                       | TCP port                                        | `8082`                                                     |
+| `etcd.port`                          | TCP port                                        | `12379`                                                     |
+| `analyzer.topology.fabric`           | Fabric connecting k8s nodes                     | `TOR1->*[Type=host]/eth0`                                  |
+| `env`                                | Extended environment variables                  | Empty                                                      |
+| `storage.elasticsearch.host`         | ElasticSearch host                              | `127.0.0.1`                                                |
+| `storage.elasticsearch.port`         | ElasticSearch port                              | `9200`                                                     |
+| `storage.flows.indicesToKeep`        | Number of flow indices to keep in storage       | `10`                                                       |
+| `storage.flows.indexEntriesLimit`    | Number of flow records to keep per index        | `10000`                                                    |
+| `storage.topology.indicesToKeep`     | Number of topology indices to keep in storage   | `10`                                                       |
+| `storage.topology.indexEntriesLimit` | Number of topology records to keep per index    | `10000`                                                    |
+| `persistence.enabled`                | Use a PVC to persist data                       | `false`                                                    |
+| `persistence.useDynamicProvisioning` | Specify a storageclass or leave empty           | `false`                                                    |
+| `dataVolume.name`                    | Name of the PVC to be created                   | `datavolume`                                               |
+| `dataVolume.existingClaimName`       | Provide an existing PersistentVolumeClaim       | `nil`                                                      |
+| `dataVolume.storageClassName`        | Storage class of backing PVC                    | `nil`                                                      |
+| `dataVolume.size`                    | Size of data volume                             | `10Gi`                                                     |
+
+## Documentation
+
+Skydive documentation can be found here:
+
+* [http://skydive-project.github.io/skydive](http://skydive-project.github.io/skydive)
+
+## Contact and Support
+
+* IRC: #skydive-project on [irc.freenode.net](https://webchat.freenode.net/)
```

Comment:
we have moved to slack

Reply:
again, this is copied from the skydive-helm readme: https://github.com/skydive-project/skydive-helm

Reply:
please correct here (and helm will have to be addressed separately)